Source/WebCore/page/ResizeObserver.h

```diff
@@ -94,8 +97,8 @@ class ResizeObserver : public RefCountedAndCanMakeWeakPtr<ResizeObserver> {
     Vector<Ref<ResizeObservation>> m_observations;
 
     Vector<Ref<ResizeObservation>> m_activeObservations;
-    Vector<GCReachableRef<Element>> m_activeObservationTargets;
-    Vector<GCReachableRef<Element>> m_targetsWaitingForFirstObservation;
+    Vector<WeakPtr<Element, WeakPtrImplWithEventTargetData>> m_activeObservationTargets;
```

Comment:
Hm... I think we need a lock. Otherwise, we can end up doing a use-after-free during `Vector` mutation.